README.md

```diff
@@ -300,10 +321,12 @@ Models for the operational use of AMIDES' misuse classification and rule attribu
 
 ## Documentation
 
-The corresponding research paper describes AMIDES in more detail:
+The corresponding research paper describes AMIDES and its evaluation results in more detail:
 
 R. Uetz, M. Herzog, L. Hackländer, S. Schwarz, and M. Henze, "You Cannot Escape Me: Detecting Evasions of SIEM Rules in Enterprise Networks,"
-in *Proceedings of the 33rd USENIX Security Symposium (USENIX Security)*, 2024. [[Conference Website](https://www.usenix.org/conference/usenixsecurity24/presentation/uetz)] [[Prepublication PDF](https://www.usenix.org/system/files/sec23winter-prepub-112-uetz.pdf)]
+in *Proceedings of the 33rd USENIX Security Symposium (USENIX Security)*, 2024. [[arXiv](https://arxiv.org/pdf/2311.10197)]
+
+Artifacts play a crucial role when it comes to availabilty and reproducability of results presented in scientific papers. The code, data, and experiments described in this repository were  submiitted to the official USENIX Artifact Evaluation (AE). The AE committee awarded the submission with all of the available badges (i.e. "Artifacts Available", "Artifacts Functional", and "Results Reproduced"). More information on the artifact submission can be found in the "Artifact Appendix" of the publication.
```

Comment:
- availabilty -> availability
- reproducability -> reproducibility
- submiitted -> submitted
- USENIX -> USENIX Security Symposium's

Reply:
Gosh. It seems like my spell checker is not working at all...